README.md

```diff
@@ -1,20 +1,102 @@
+
+## Project structuring
+
+The project is structured in 3 parts which consist of the server, test-client, and client
+
+### Server
+
+The server structure consists of all the code that allows the bot to communicate with the users effectively. Here is a detailed explanation of each substructure in the server.
+
+1.  Controllers —> This folder consist of the code handler that determines if the user's input is a question or a spec related input
+2. Helpers —>  This folder consists of all the external modules the bot is using for communication. E.g. wit.js(for communicating with the [wit.ai](http://wit.ai) NLP) and socket.js(used for real-time communication between the bot and users)
+3. Models —> This folder consists of all data required by the bot. E.g the question models which consist of all the questions the bot asks the user. 
+4. Services —> This folder consists of codes that process the user's input. E.g the generatorFlow.js(this handles all the spec generation-related inputs), questionFlow.js(this handles all question-related inputs), and generator.js(this handles the generation of the final asyncapi specification in a YAML format
+5. Utils —> This folder consists of all reusable codes used across the bots server-side e.g schema validation
+
+### Test Client
+
+This is an Interactive CLI that can be used to interact with the bot... Nothing much here
+
+### Client
+
+This folder consists of all interactive User interface code that we'll be used to interact with the bot from the browser [Coming soon] 
+
+## Common bots vocabulary
+
+1. Skip or Not now —> To tell the bot to skip a question
+2. Exit or Kill—> To tell the bot to end the conversation [Not available yet]
+3. ? —> To tell the bot to repeat the question [Not available yet]
+
+# Project's Todo
+
+---
+
+- [ ]  An interactive user interface that can be plugged into the asycapi website, playground, and the upcoming studio
+- [ ]  Ability to readout specs and give info about channels that users ask for from the spec
+- [ ]  Handle and understand more vocabulary
+- [ ]  Ability to answer more asyncapi related questions
+- [ ]  Ability to modify an existing spec by parsing an existing spec link. So users can tell the bot they want to modify the specification
+
+# Getting started
+
+---
+
+### Prerequisite
+
+1. Git
+2. Node: any 12.x version starting with v12.0.0 or greater
+3. Yarn
+4. A fork of the repo (for any contributions)
+5. A clone of the [chatbots](https://github.com/asyncapi/chatbot) repo on your local machine
+
+### Installation
+
+1. cd [chatbots](https://github.com/asyncapi/chatbot) to go into the project root
+2. yarn to install the dependencies
+
+### Running locally
+
+Follow this step to run the project with the live server
+
+1. cd into packages/test-client
+2. create a .env and add PORT=[http://167.71.46.87/](http://167.71.46.87/) API_KEY="reach out to the maintainers for an API key" to communicate with the live server.  
```

Comment:
please mention that somebody should rather get their own Wit account and train their own model, we should not share API Key 😅

Reply:
also where is this IP coming from? isn't better to do `ACCESS_TOKEN=abs yarn start`? we should make it as easy to start as possible

Reply:
Can you explain better what you mean?

Reply:
> you can pass environment variables on runtime in terminal as `ACCESS_TOKEN=abs yarn start`, no need to edit .env file
> 


Okay

Reply:
> also, the IP that you have in this instruction, what is it for? where is the IP coming from?

It's the IP address for the live server

Reply:
oh, you mean you managed to deploy it? perfect! So the API_KEY is not for Wit.ai but for the deployed application?

also if it is IP then why it is called PORT? maybe ADDRESS?